roles/contiv_network/tasks/aci_tasks.yml

```diff
@@ -0,0 +1,49 @@
+# These tasks are run when aci_mode is set
+
+- name: check aci-gw container image
+  shell: docker inspect contiv/aci-gw
+  register: docker_aci_inspect_result
+  ignore_errors: yes
+
+- name: pull aci-gw container
+  shell: docker pull contiv/aci-gw
+  when: "'No such image' in docker_aci_inspect_result.stderr"
+  run_once: true
+
+- name: check if aci-gw container is already running
+  shell: docker ps | grep -sq 'contiv/aci-gw'
+  register: aci_running
+  changed_when: false
+  failed_when: false
+  run_once: true
+
+- name: start aci-gw container
+  shell: >
+    docker run -itd --net=host
+    -e "APIC_URL: "{{ apic_url }}
+    -e "APIC_USERNAME: "{{ apic_username }}
+    -e "APIC_PASSWORD: "{{ apic_password }}
+    -e "APIC_LEAF_NODE: "{{ apic_leaf_nodes }}
+    --restart=unless-stopped
+    --name=contiv-aci-gw
+    contiv/aci-gw
+  when: aci_running.rc != 0
+  run_once: true
+  ignore_errors: yes
+
+- name: set aci mode
+  shell: contivctl global set -nwinfra aci
+  run_once: true
+
+#NOTE: These tasks need to be modified once there is release-version for contiv/libcompose
+- name: download contiv version of docker-compose
+  get_url:
+    validate_certs: "{{ validate_certs }}"
+    url: https://cisco.box.com/shared/static/eagplmveetdmwn7u2uwadna07l1m8t3s.bz2
+    dest: /tmp/contiv-compose.bz2
```

Comment:
Is this built from `contiv/libcompose`?  I think it's same as `docker-compose` so it should have a version flag already.